src/scripts/modules/Hello.ts

```diff
@@ -3,6 +3,6 @@ import { getNode } from './utils';
 
 export const sayHello = () => {
```

Comment:
It would be nice if the name of function was changed to logHelloWorld(). Can you amend it?